Flex logs callout for compute limitation (#33877)
DataDog/documentation: content/en/logs/log_configuration/flex_logs.md

### Flex Logs for multiple-organization accounts

<div class="alert alert-danger">Each organization can only use one compute size at a time. Compute sizes cannot be shared across organizations, and Starter and scalable compute cannot be used simultaneously within the same organization.</div>

> For each organization in which you want Flex Logs, you must enable a compute size per organization. Datadog recommends Flex Logs scalable compute sizes (XS, S, M, and L) for organizations with large log volumes. In a multi-organization setup, there are often many organizations with lower log volumes, so for these organizations, Datadog recommends the Starter compute size for Flex Logs.

Reviewer: Slight adjustment for flow. Also cutting out the "per organization" at the end since we have "for each org" up top!

```suggestion
For each organization where you want to use Flex Logs, you must enable a compute size. Datadog recommends Flex Logs scalable compute sizes (XS, S, M, and L) for organizations with large log volumes. In a multi-organization setup, there are often many organizations with lower log volumes, so for these organizations, Datadog recommends the Starter compute size for Flex Logs.
```
